src/Datadog.Trace.ClrProfiler.Managed/AutoInstrumentation/Msmq/MessageQueue_ReceiveCurrent_Integration.cs

```diff
@@ -0,0 +1,67 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Datadog.Trace.ClrProfiler.CallTarget;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.DuckTyping;
+using Datadog.Trace.Logging;
+using Datadog.Trace.Tagging;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Msmq
+{
+    /// <summary>
+    /// Msmq calltarget instrumentation
+    /// </summary>
+    [InstrumentMethod(
+    AssemblyName = "System.Messaging",
+    TypeName = "System.Messaging.MessageQueue",
+    MethodName = "ReceiveCurrent",
+    ReturnTypeName = ClrNames.MsmqMessage,
+    ParameterTypeNames = new[] { ClrNames.TimeSpan, ClrNames.Int32, ClrNames.MsmqCursorHandle, ClrNames.MsmqMessagePropertyFilter, ClrNames.MsmqMessageQueueTransaction, ClrNames.MsmqMessageQueueTransactionType },
+    MinimumVersion = "4.0.0",
+    MaximumVersion = "4.*.*",
+    IntegrationName = IntegrationName)]
+    public class MessageQueue_ReceiveCurrent_Integration
+    {
+        private const string CommandPeek = "msmq.peek";
+        private const string CommandConsume = "msmq.consume";
+        private const string IntegrationName = nameof(IntegrationIds.Msmq);
```

Comment:
NIT: you could move them to `MsmqConstants` (same for similar constants in other integration files)